lib/icingadb/icingadb.ti

```diff
@@ -10,7 +10,7 @@ namespace icinga
 
 class IcingaDB : ConfigObject
 {
-	activation_priority 100;
+	activation_priority 250;
```

Comment:
> All the data outputs (and externalcommandlistener.ti which is rather accidentally there and I'd not touch now) have a harmonized AP of 100. I'd like to keep it harmonized. Btw. let's literally keep 100 to keep the diff small. If you need data outputs later than notificationcomponent.ti, just lower the latter AP and keep the 100-s.

Reply:
The problem is that NotificationComponent has a activation_priority of 200, which means, there is some time, where notifications are already/still being sent, but the event is not written into the db. I'd like to avoid that entirely.

Reply:
Then lower NotificationComponent's activation_priority below 100 instead.